### DISCUSSION TITLE
Paragraph writing

### DISCUSSION THREAD

**Discussion by Zoe-Elizabeth91** [community/community#129736] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello 

I want to write titles and paragraphs on my github project like in the picture below but im a beginner and im not sure how to do it. I have already added my python project but I want to add the titles and paragraphs. Could someone help me please? Thank you. 
![image](https://github.com/community/community/assets/173475679/81cd0d2a-2a6a-46e5-ba14-575af224926c)

**Comment by m1her** [comment]
Add README.md file to the repo, and this could help you formatting its [syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

**Comment by Mayur-Pagote** [comment]
@Zoe-Elizabeth91, I see that you want to add titles and paragraphs to your GitHub project by creating a `README.md` file. To format it, you need to be familiar with Markdown or HTML. 

For example, you can use `#` to mark a heading and `##` or `###` for subheadings. Here's an example:

```markdown
# This is a title
## This is a subtitle
```

This will render as:  
# This is a title  
## This is a subtitle  

Hope this will help you.
